enhancements/ingress/ingress-dashboard.md

```diff
@@ -0,0 +1,154 @@
+---
+title: ingress-dashboard
+authors:
+  - "@jotak"
+reviewers:
+  - "@Miciah"
+  - "@candita"
+  - "@OlivierCazade"
+approvers:
+  - "@Miciah"
+  - "@candita"
```

Comment:
@Miciah @candita I set you as both reviewers and approvers, let me know if that's ok / if I should add more people

Reply:
@jotak please add the epic number to the title, and a brief description.